.buildkite/dagster-buildkite/dagster_buildkite/utils.py

```diff
@@ -94,3 +96,13 @@ def connect_sibling_docker_container(network_name, container_name, env_variable)
 
 def is_release_branch(branch_name: str):
     return branch_name.startswith("release-")
+
+
+def get_python_versions():
```

Comment:
I think we should name this to make it clear its behavior deviates based on branch